grpc/applist.proto

```diff
@@ -220,6 +220,7 @@ message ReleaseIdentifier {
 message UpgradeReleaseRequest {
   ReleaseIdentifier releaseIdentifier = 1;
   string valuesYaml = 2;
+  int32 revisionMaxLimit = 3;
```

Comment:
keep it consistent with helm flag `history-max`

Reply:
done